spec/controllers/education_experiences_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EducationExperiencesController, type: :controller do
+  render_views
+
+
+
+  describe 'GET #index' do
+    let(:user) { create(:user) }
+
+    let(:page) { Capybara::Node::Simple.new(response.body) }
+
+    context 'when the user is logged in' do
+      before do
+        sign_in user
+      end
+
+      it 'renders the index template' do
+        get :index
+
+        expect(response).to render_template(:index)
+      end
+
+      it 'has status 200' do
+        get :index
+
+        expect(response).to have_http_status(:ok)
+      end
+
+      context 'when the user does not contain education experiences' do
+        it 'renders the table with not found message' do
+          get :index
+
+          expect(page.find('table')).to have_content(I18n.t('resumes.education_experience.not_found'))
```

Comment:
use the text directly in the test, this way if the translation has a typo or is wrong, is easier to notice